api-matlab/rtQC_display.m

```diff
@@ -1456,6 +1464,36 @@ function generateReport(hObject,eventdata)
 
 end
 
+function physioBatch(hObject,eventdata)
+    % check if physio is installed:
+    pathCell = regexp(path, pathsep, 'split');
+    if ispc  % Windows is not case-sensitive
+      onPath = any(contains(pathCell, 'PhysIO'));
+    else
+      onPath = any(contains(pathCell, 'PhysIO'));
+    end
+    if ~onPath
+        error("ERROR: Physio toolbox not installed or not added to path. Follow installation instructions on https://github.com/translationalneuromodeling/tapas.");    
```

Comment:
The double quote gives an error in Matlab versions earlier than 2017a, I suggest we use single quotes for now and then add code to handle version differences in a future PR